src/Data/List/Relation/Unary/Any.agda

```diff
@@ -69,7 +69,7 @@ _∷=_ {xs = xs} x∈xs v = xs List.[ index x∈xs ]∷= v
 
 infixl 4 _─_
 _─_ : {P : Pred A p} → ∀ xs → Any P xs → List A
-xs ─ x∈xs = xs List.─ index x∈xs
+xs ─ x∈xs = removeAt xs (index x∈xs)
```

Comment:
Should this operation `_─_` not also be renamed to `removeAt`... if so, you'll need to remove that name from the `import` list for `List`, and use `List.removeAt` on the RHS, ...